fast_galaxy/fast_galaxy.py

```diff
@@ -39,9 +39,12 @@ def split_requirements(file_path):
     print(f"Split requirements file into individual role and collection files in the '{role_files_dir}' directory.")
     return role_files + collection_files
 
-def install_requirements(file_path):
+def install_requirements(file_path, force=False):
     try:
-        result = subprocess.run(["ansible-galaxy", "install", "-r", file_path], check=True, capture_output=True, text=True)
+        if force:
+            result = subprocess.run(["ansible-galaxy", "install", "--force", "-r", file_path], check=True, capture_output=True, text=True)
+        else:
+            result = subprocess.run(["ansible-galaxy", "install", "-r", file_path], check=True, capture_output=True, text=True)
```

Comment:
I would better do the following to factorize the execution:
```suggestion
        command_line = ["ansible-galaxy", "install", "-r", file_path]
        if force:
            command_line.append("--force")
        result = subprocess.run(command_line, check=True, capture_output=True, text=True)
```

Reply:
Fine for me.